c/algorithm/BubbleSort.c

```diff
@@ -0,0 +1,11 @@
+static void BubbleSort(int[] arr){
```

Comment:
You need to pass the size of the array, let say `unsigned int n` so the signature would become:
```c
static void BubbleSort(int[] arr, unsigned int n) {
```